src/model/markercollection.js

```diff
@@ -129,6 +129,27 @@ export default class MarkerCollection {
 	}
 
 	/**
+	 * Iterates over all markers that starts with given `prefix`.
+	 *
+	 *		const markerFooA = markersCollection.set( 'foo:a', rangeFooA );
+	 *		const markerFooB = markersCollection.set( 'foo:b', rangeFooB );
+	 *		const markerBarA = markersCollection.set( 'bar:a', rangeBarA );
+	 *		const markerFooBarA = markersCollection.set( 'foobar:a', rangeFooBarA );
+	 *		Array.from( markersCollection.getMarkersGroup( 'foo' ) ); // [ markerFooA, markerFooB ]
+	 *		Array.from( markersCollection.getMarkersGroup( 'a' ) ); // []
+	 *
+	 * @param prefix
+	 * @returns {Iterator.<~Marker>}
+	 */
+	*getMarkersGroup( prefix ) {
+		for ( let marker of this._markers.values() ) {
+			if ( marker.name.indexOf( prefix + ':' ) === 0 ) {
```

Comment:
Did you consider using startsWith? https://developer.mozilla.org/pl/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith